engine/engine/src/engine.cpp

```diff
@@ -1009,10 +1009,10 @@ namespace dmEngine
         }
 
         // rig.max_instance_count is deprecated in favour of component specific max count values.
-        // For backwards combatibility we get the rig generic value and take the max of it and each
-        // specific component max value.
-        int32_t max_rig_instance = dmConfigFile::GetInt(engine->m_Config, "rig.max_instance_count", 128);
-        int32_t max_model_count = dmMath::Max(dmConfigFile::GetInt(engine->m_Config, "model.max_count", 128), max_rig_instance);
+        if (dmConfigFile::GetInt(engine->m_Config, "rig.max_instance_count", -1) != -1)
+        {
+            dmLogWarning("`rig.max_instance_count` deprecated. Use component specific counters.");
```

Comment:
Could we name the specific counters that can be used?

Reply:
for now it's model.max_counter
But it depends on when we decde to use rig in the future, so I'm not sure if we wanna hardcode it here